sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -1338,5 +1343,112 @@ describe(
       });
       expect(nonUpdatedCSLPOveraward).toBe(true);
     });
+
+    it("Should Stop disburse BC funding for a part time application when a restriction is applied.", async () => {
+      // Arrange
+      // Eligible COE basic properties.
+      const eligibleDisbursement: Partial<DisbursementSchedule> = {
+        coeStatus: COEStatus.completed,
+        coeUpdatedAt: new Date(),
+      };
+      // Create a B6A restriction.
+      const b6aRestriction = await db.restriction.findOneBy({
+        restrictionCode: RestrictionCode.B6A,
+      });
```

Comment:
No need to be specific with the restriction if we find it as below (code from the same file).
No a blocker but it would prevent dealing with the restriction code directly since the E2E test is about the `StopPartTimeBCFunding` and not about a specific restriction.
```ts
const restriction = await db.restriction.findOne({
  where: {
    actionType: ArrayContains([
      RestrictionActionType.StopPartTimeBCFunding,
    ]),
  },
});
```

Reply:
Thanks for making the change.
@guru-aot I am approving the PR since this entire comment was just a suggestion even though I saw value in your suggested changed.